### PR TITLE
VACMS-2021: Update workflow states and transitions.

### PR DESCRIPTION
**Pull request by oksana-c**
## Description

See #2021 . 

## Screenshots

Screen Shot 2020-07-17 at 7 03 29 AM

## QA steps

- [ ] visit `/admin/config/workflow/workflows/manage/editorial`
- confirm the following
  - [ ]  `Approved` state is added
  - [ ]  `Approve` transition is added: FROM state `In Review`, TO state `Approved`
  - [ ] `Create New Draft` transition is renamed to `Edit`
  - [ ] `Send to Review` transition is renamed to `Review`

NOTE: adjustments to permissions per transition are not in scope of #2021

### CODE REVIEW

**Comment by oksana-c** [comment]
@indytechcook , tagging you as a reviewer on this one, so that you're in the loop on all things EWA

File: tests/behat/drupal-spec-tool/workflow.feature [hunk 1]
```diff
@@ -13,24 +13,26 @@ Feature: Workflow
   @spec @dst @workflow_states
   Scenario: Workflow states
     Then exactly the following workflow states should exist
-      | Workflow           | Label     | Machine name         |
-      | Editorial workflow | Draft     | draft                |
-      | Editorial workflow | In review | review               |
-      | Editorial workflow | Published | published            |
-      | Editorial workflow | Archived  | archived             |
+      | Workflow | Label | Machine name |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/workflow.feature:16]: Can you set your editor to not cleanup the tests as they come from spectool.  It makes them a lot harder to read and will flip flop with the others who are not cleaning up spec paste.

Comment (oksana-c) [reply tests/behat/drupal-spec-tool/workflow.feature:16]: good point. I'll look into this. btw, let's keep this PR hanging for now until I get a moment to set IDE config -  its not urgent to merge

Comment (oksana-c) [reply tests/behat/drupal-spec-tool/workflow.feature]: to follow up on formatting. I've checked and don't believe it's my IDE that's removing extra spaces. As far as I can see, the spec tool does not provide them. Maybe I'm overlooking something though.
I agree that we need to make an effort to keep `.feature` files more readable, but am also wondering if manual adjustments after pasting is the way to go.
Let me know when you have a minute to chat about this.

**Pull request merged**